Source/relay/relay_TourGuide.ash

```diff
@@ -1,6 +1,6 @@
 //This script and its support scripts are in the public domain.
 
-since r26713; // $path update
+since r27273; // 8BitBonusTurns implemented
```

Comment:
this now represents a conflict after merging your other newbies. can you merge in main and then re-update so this is mergable?